projects/packages/connection/src/sso/class-user-admin.php

```diff
@@ -261,19 +265,27 @@ public function invite_user_to_wpcom() {
 			);
 
 			if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
-				$error        = 'invalid-invite-api-error';
+				$error_code   = 'invalid-invite-api-error';
```

Comment:
Wouldn't it be useful if `$error_code` was also set from the response, instead of using a generic error code?